doc/flatpak-manifest.xml

```diff
@@ -389,7 +389,7 @@
                 </varlistentry>
                 <varlistentry>
                     <term><option>config-opts</option> (array of strings)</term>
-                    <listitem><para>This is an array containing extra options to pass to configure.</para></listitem>
+                    <listitem><para>This is an array containing extra options to pass to configure or cmake.</para></listitem>
```

Comment:
```suggestion
                    <listitem><para>This is an array containing extra options passed to the build system during configuration.</para></listitem>
```

Reply:
This applies to all build systems, is this still clear to you?

Reply:
Thanks - the docs made it looks like it only applied if running `configure`, and I found by trial and error that it applies to cmake too. Your new phrasing avoids the implication that it only applies if there's a `configure` script, thanks.